glide.yaml

```diff
@@ -0,0 +1,227 @@
+  - vendor/github.com/pingcap/kvproto/pkg/msgpb
+  - vendor/github.com/pingcap/kvproto/pkg/pdpb
+  - vendor/github.com/pingcap/kvproto/pkg/util
+  - pkg/metrics
+  - vendor/github.com/prometheus/client_golang/prometheus
+  - vendor/github.com/twinj/uuid
+  - vendor/golang.org/x/net/context
+  - vendor/github.com/coreos/etcd/auth/authpb
+  - vendor/github.com/coreos/etcd/etcdserver/api/v3rpc/rpctypes
+  - vendor/github.com/coreos/etcd/etcdserver/etcdserverpb
+  - vendor/github.com/coreos/etcd/mvcc/mvccpb
+  - vendor/github.com/coreos/etcd/pkg/tlsutil
+  - vendor/github.com/ghodss/yaml
+  - vendor/google.golang.org/grpc
+  - vendor/google.golang.org/grpc/codes
+  - vendor/google.golang.org/grpc/credentials
+  - vendor/google.golang.org/grpc/grpclog
+  - vendor/google.golang.org/grpc/metadata
+  - vendor/github.com/golang/protobuf/proto
+  - vendor/github.com/pingcap/kvproto/pkg/coprocessor
+  - vendor/github.com/pingcap/kvproto/pkg/kvrpcpb
+  - vendor/github.com/pingcap/kvproto/pkg/raft_cmdpb
+  - vendor/github.com/pingcap/kvproto/pkg/raft_serverpb
+  - vendor/github.com/pingcap/kvproto/pkg/eraftpb
+  - vendor/github.com/beorn7/perks/quantile
+  - vendor/github.com/prometheus/client_model/go
+  - vendor/github.com/prometheus/common/expfmt
+  - vendor/github.com/prometheus/procfs
+  - vendor/github.com/grpc-ecosystem/grpc-gateway/runtime
+  - vendor/github.com/grpc-ecosystem/grpc-gateway/utilities
+  - vendor/github.com/cloudfoundry-incubator/candiedyaml
+  - vendor/golang.org/x/net/http2
+  - vendor/golang.org/x/net/trace
+  - vendor/google.golang.org/grpc/internal
+  - vendor/google.golang.org/grpc/naming
+  - vendor/google.golang.org/grpc/transport
+  - vendor/github.com/pingcap/kvproto/pkg/errorpb
+  - vendor/github.com/matttproud/golang_protobuf_extensions/pbutil
+  - vendor/github.com/prometheus/common/internal/bitbucket.org/ww/goautoneg
+  - vendor/github.com/prometheus/common/model
+  - vendor/github.com/golang/protobuf/jsonpb
+  - vendor/github.com/grpc-ecosystem/grpc-gateway/runtime/internal
+  - vendor/golang.org/x/net/http2/hpack
+  - vendor/golang.org/x/net/lex/httplex
+  - vendor/golang.org/x/net/internal/timeseries
+  - vendor/google.golang.org/grpc/peer
+- package: github.com/pingcap/tipb
+  version: 076169255d6d516f4a72d1648c085d872ff8bf48
+  subpackages:
+  - go-binlog
+  - go-tipb
+- package: github.com/twinj/uuid
+  version: 70cac2bcd273ef6a371bb96cde363d28b68734c3
+- package: golang.org/x/net
+  version: b797637b7aeeed133049c7281bfa31dcc9ca42d6
+  subpackages:
+  - context
+  - http2
+  - http2/hpack
+  - internal/timeseries
+  - trace
+- package: golang.org/x/text
+  version: 723492b65e225eafcba054e76ba18bb9c5ac1ea2
+  subpackages:
+  - encoding
+  - encoding/charmap
+  - encoding/internal
+  - encoding/internal/identifier
+  - encoding/japanese
+  - encoding/korean
+  - encoding/simplifiedchinese
+  - encoding/traditionalchinese
+  - encoding/unicode
+  - transform
+- package: google.golang.org/grpc
```

Comment:
If `glide update` failed, please refer to https://github.com/pingcap/pd/blob/master/glide.yaml#L185